Sources/Clappr/Classes/Base/Loader.swift

```diff
@@ -18,25 +16,24 @@ open class Loader {
     public init(externalPlugins: [Plugin.Type], options: Options = [:]) {
         self.externalPlugins = externalPlugins
 
-        loadExternalMediaControl(options)
-
+        #if os(iOS)
+        addMediaControl()
+        #endif
+        
         if !externalPlugins.isEmpty {
             addExternalPlugins(externalPlugins)
         }
 
         Logger.logInfo("plugins:" +
             "\n - playback: \(playbackPlugins.map({ $0.name }))" +
             "\n - container: \(containerPlugins.map({ $0.name }))" +
-            "\n - core: \(corePlugins.map({ $0.name }))" +
-            "\n - mediaControl: \(mediaControl)", scope: "\(type(of: self))")
+            "\n - core: \(corePlugins.map({ $0.name }))", scope: "\(type(of: self))")
     }
 
-    fileprivate func loadExternalMediaControl(_ options: Options) {
```

Comment:
I believe that we've removed a Clappr feature: to pass your own `MediaControl`.

If this change goes to production all of the customized `MediaControl` will stop to work. Bringing this feature back would be as simples as return these deleted lines, right?

What do you guys think?